breakdown.go

```diff
@@ -66,6 +66,11 @@ type spanTimingsKey struct {
 	spanSubtype string
 }
 
+type droppedSpanTimingsKey struct {
+	destination string
+	outcome     string
+}
```

Comment:
This isn't related to breakdown metrics. I think it probably belongs next to `droppedSpanTimingsMap`, in transaction.go